### DISCUSSION TITLE
Argument of type 'number' is not assignable to parameter of type 'APIResponse'.ts(2345)

### DISCUSSION THREAD

**Discussion by AutoQA23** [akshayp7/playwright-typescript-playwright-test#48] [Q&A] [answered]
Hi @akshayp7 ,
I am getting error ts(2345) as API response is not accepting the number. Please help.

const response = await request.get(`/api/users?per_page=1`);
    await apiActions.verifyStatusCode(response.status(), 500);

  async verifyStatusCode(response: APIResponse): Promise<void> {
        await expect(response, `200 Status code was not displayed.`).toBeOK();
    }

**Comment by akshayp7** [comment, marked as answer]
response.status() . Try this if it accepts number.
